public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -49,6 +49,36 @@ export function PluginDetailsBody({ plugin, queryParams, pageId }: Props): JSX.E
     );
   }
 
+  if (pageId === PluginTabIds.IAM) {
+    return (
+      <div className={styles.container}>
+        <VerticalGroup>
```

Comment:
@Clarity-89 I'm using `VerticalGroup` and `HorizontalGroup` here, is it fine? Should I remove the outer `div`?

Reply:
We now have Stack component, which we recommend using instead of the Vertical and Horizontal groups. `<Stack direction='column'/>` is equivalent to `VerticalGroup`. Also yes, container shouldn't be necessary here.